roles/ansible-test-splitter/files/list_changed_targets.py

```diff
@@ -412,14 +435,69 @@ def build_result_struct(self, batches):
         return result
 
 
+def read_pullrequest_body(project_name, pr_number):
+    try:
+
+        if HAS_GITHUB_MODULE:
+            g = Github()
+            repo = g.get_repo(project_name)
+            pr = repo.get_pull(pr_number)
+            return [x for x in pr.body.split("\n") if x]
+
+    except:
+        return []
+
+
+def read_user_extra_requests(project_name, pr_number):
+
+    desc = read_pullrequest_body(project_name, pr_number)
+    ansible_test_regex = (
```

Comment:
Those are not regex, I would call them `extra_requests_patterns` instead to avoid the confusion.